batch/batch/worker.py

```diff
@@ -37,6 +37,9 @@
 log = logging.getLogger('batch-worker')
 
 MAX_IDLE_TIME_MSECS = 30 * 1000
+MAX_DOCKER_IMAGE_PULL_SECS = 20 * 60
+MAX_DOCKER_WAIT_SECS = 5 * 60
+MAX_DOCKER_TIMEOUT_SECS = 1 * 60
```

Comment:
using `MAX` and `TIMEOUT` in one variable name seems wrong. I prefer:

```
MAX_DOCKER_IMAGE_PULL_SECS
MAX_DOCKER_WAIT_SECS
MAX_DOCKER_OTHER_OPERATION_SECS
```

Reply:
And if someone sees the last one out of this context they'll understand that this is some kind of catch-all and that there might be other timeouts.